fastn-core/src/library2022/processor/sqlite.rs

```diff
@@ -33,6 +33,44 @@ pub async fn process(
 ) -> ftd::interpreter::Result<ftd::interpreter::Value> {
     let (headers, query) = get_p1_data("package-data", &value, doc.name)?;
 
+    let use_db = match headers.get_optional_string_by_key("use", doc.name, value.line_number()) {
+        Ok(Some(k)) => Some(k),
+        _ => match headers
+            .get_optional_string_by_key("db", doc.name, value.line_number())
+            .ok()
+        {
+            Some(k) => {
+                println!("Warning: `db` header is deprecated, use `use` instead.");
+                k
+            }
```

Comment:
Lets just use `db` header for this PR, we are not so finalised in the use of `use`. We can make this change, if finalised, later.